src/components/MyNavbar/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import logo from '../../img/logo.svg';
+import 'font-awesome/css/font-awesome.min.css';
+import './style.css';
+import { Nav, Navbar } from "react-bootstrap";
+
+export default function MyNavbar() {
```

Comment:
Renamed to `MyNavbar` to avoid the name clash with react-bootstrap's `Navbar` type.